employees/forms.py

```diff
@@ -57,7 +57,11 @@ class Meta:
 
     def __init__(self, *args: Any, **kwargs: Any):
         super(ReportForm, self).__init__(*args, **kwargs)
-        self.fields["project"].queryset = kwargs["initial"]["author"].projects.all()
+        author = kwargs["initial"]["author"]
```

Comment:
What do you think about making `author` a required parameter for this `init`? I think it should be always present, otherwise this will fail easily. If you want I can add it in https://github.com/Code-Poets/sheetstorm/pull/327.

Reply:
ok, please add it and I will rebase